libs/utils/src/format.ts

```diff
@@ -3,6 +3,10 @@ const countFormatter = new Intl.NumberFormat(undefined, { useGrouping: true })
 /**
  * Format integers for display as whole numbers, i.e. a count of something.
  */
-export function count(value: number): string {
```

Comment:
I'd actually keep this as-is and change `index.ts` to `export * as format from './format'`.